tests/files/tests.py

```diff
@@ -169,6 +169,25 @@ def test_io_wrapper(self):
             test_file.seek(0)
             self.assertEqual(test_file.read(), (content * 2).encode())
 
+    @unittest.skipIf(os.name == 'nt', "Posix specific tests.")
+    def test_file_lock(self):
+        file_path = os.path.join(os.path.dirname(__file__), "test.png")
+        with open(file_path, 'rb') as f:
+            self.assertIs(locks.lock(f, locks.LOCK_EX), True)
+
+        with open(file_path, 'rb') as f:
+            self.assertIs(locks.lock(f, locks.LOCK_NB), False)
```

Comment:
You need to specify a type of lock, e.g. `locks.LOCK_NB | locks.LOCK_EX`. Currently it returns `False` because we passed an invalid argument.